constructor/nsis/main.nsi.tmpl

```diff
@@ -550,13 +545,13 @@ Function .onInit
     Push $R2
 
     InitPluginsDir
-    @TEMP_EXTRA_FILES@
+    {{ TEMP_EXTRA_FILES }}
     !insertmacro ParseCommandLineArgs
 
     # Select the correct registry to look at, depending
     # on whether it's a 32-bit or 64-bit installer
-    SetRegView @BITS@
-#if win64
+    SetRegView {{ BITS }}
+{%- if win64 %}
```

Comment:
Do we want to deprecate win-32 altogether?

Reply:
Let's do that in another PR for better traceability